src/components/pages/SideDrawer/SideDrawer.js

```diff
@@ -33,42 +22,58 @@ function SideDrawer() {
   };
 
   return (
-    <div style={{ width: 256 }}>
-      <Button
-        type="primary"
-        onClick={toggleCollapsed}
-        style={{ marginBottom: 16 }}
-      >
-        {React.createElement(collapsed ? MenuUnfoldOutlined : MenuFoldOutlined)}
-      </Button>
-
+    <div style={{ width: 246 }}>
       <Menu
         defaultSelectedKeys={['none']}
-        defaultOpenKeys={['saved', 'admin-tools']}
         mode="inline"
-        theme="dark"
+        theme="light"
```

Comment:
Great job refactoring these components and changing the theme to better match overall styling!